kolibri/content/serializers.py

```diff
@@ -0,0 +1,20 @@
+from models import ContentMetadata, File, Format
+from rest_framework import serializers
+
+
+class ContentMetadataSerializer(serializers.ModelSerializer):
```

Comment:
Oh no, serializer is indeed database specific, it has a save() function that will operate on models